apps/golden-retriver/src/app.controller.ts

```diff
@@ -1,10 +1,20 @@
 import { Controller, Get } from '@nestjs/common';
```

Comment:
make sure ur pr uses this template before requesting review https://github.com/Pawmery-pet/pet-turbo/blob/develop/.github/pull_request_template.md